modules/machineset-yaml-aws.adoc

```diff
@@ -188,26 +188,25 @@ $ oc -n openshift-machine-api \
     -o jsonpath='{.spec.template.spec.providerSpec.value.ami.id}{"\n"}' \
     get machineset/<infrastructure_id>-<role>-<zone>
 ----
-<5> Optional: Specify custom tag data for your cluster. For example, you might add an admin contact email address by specifying a `name:value` pair of `Email:\admin-email@example.com`.
-+
-[NOTE]
-====
-Custom tags can also be specified during installation in the `install-config.yml` file. If the `install-config.yml` file and the machine set include a tag with the same `name` data, the value for the tag from the machine set takes priority over the value for the tag in the `install-config.yml` file.
-====
-
 ifndef::edge[]
-<6> Specify the zone, for example, `us-east-1a`.
+<5> Specify the zone name, for example, `us-east-1a`.
 endif::edge[]
 ifdef::edge[]
-<6> Specify the zone name, for example, `us-east-1-nyc-1a`.
-endif::edge[]
-<7> Specify the region, for example, `us-east-1`.
+<5> Specify the zone name, for example, `us-east-1-nyc-1a`.
+endif::edge[] 
+<6> Specify the region, for example, `us-east-1`.
 ifndef::edge[]
-<8> Specify the infrastructure ID and zone.
+<7> Specify the infrastructure ID and zone.
 endif::edge[]
 ifdef::edge[]
-<8> The ID of the public subnet that you created in AWS {zone-type}. You created this public subnet ID when you finished the procedure for "Creating a subnet in an AWS zone".
+<7> The ID of the public subnet that you created in AWS {zone-type}. You created this public subnet ID when you finished the procedure for "Creating a subnet in an AWS zone".
 endif::edge[]
+<8> Optional: Specify custom tag data for your cluster. For example, you might add an admin contact email address by specifying a `name:value` pair of `Email:\admin-email@example.com`.
++
+[NOTE]
+====
+Custom tags can also be specified during installation in the `install-config.yml` file. If the `install-config.yml` file and the machine set include a tag with the same `name` data, the value for the tag from the machine set takes priority over the value for the tag in the `install-config.yml` file.
```

Comment:
```suggestion
You can also specify custom tags during installation in the `install-config.yaml` file. If the `install-config.yml` file and the machine set include a tag with the same `name` data, the value for the tag from the machine set takes priority over the value for the tag in the `install-config.yaml` file.
```

- Active voice can be used in the note.
- Replace `install-config.yml` / `install-config.yaml`